app/Repositories/SignupRepository.php

```diff
@@ -44,6 +45,24 @@ public function create($data)
     }
 
     /**
+     * Update a signup. Quanity and Why participated are the
+     * only fields that can be updated, at the moment.
+     *
+     * @param  Rogue\Models\Signup $signup
+     * @param  array $data
+     * @return \Rogue\Models\Signup|null
+     */
+    public function update($signup, $data)
+    {
+        $signup->quantity = isset($data['quantity']) ? $data['quantity'] : $signup->quantity;
```

Comment:
oh i see! i wasn't doing mass-assigment with `update` because I wanted to make sure the model event triggered but see that `fill` is another option!